comparison.html

```diff
@@ -1,15 +1,56 @@
 <h1>Comparison to Other DID Methods</h1>
 
-<p>We could certainly build peer relationships with <a>anywise</a> DIDs based on a public ledger or a similar
-    source of truth. However, in the same way that a company doesn't want the public to resolve private host
-    names inside its corporate intranet, letting others resolve <a>pairwise</a> and <a>n-wise</a> DIDs is
-    unnecessary, and it represents a privacy and security risk as well as a problem of cost, scale, and
-    performance. We strongly recommend that peer DIDs be used for peer relationships.
-</p>
-<p>In a similar vein, peer DIDs could be used, hypothetically, for <a>anywise</a> scenarios. The main disadvantage
-    would be the lack of a formal publication mechanism. Nothing would prevent a user from publishing a peer DID and
-    its associated DID document on a website. However, information published in this way would be hard to discover,
-    maintain as DID docs evolved, and integrate into interoperable applications. DID methods that use
-    a public ledger or a similar source of truth are a better choice here, because they have authoritative
-    answers to the publication problem.
-</p>
+<section>
+    <h2>Ledger-oriented Methods</h2>
+    <p>We could certainly build peer relationships with <a>anywise</a> DIDs based on a public ledger or a similar
+        source of truth. However, in the same way that a company doesn't want the public to resolve private host
+        names inside its corporate intranet, letting others resolve <a>pairwise</a> and <a>n-wise</a> DIDs is
+        unnecessary, and it represents a privacy and security risk as well as a problem of cost, scale, and
+        performance. We strongly recommend that peer DIDs be used for peer relationships.
+    </p>
+    <p>In a similar vein, peer DIDs could be used, hypothetically, for <a>anywise</a> scenarios. The main disadvantage
+        would be the lack of a formal publication mechanism. Nothing would prevent a user from publishing a peer DID and
+        its associated DID document on a website. However, information published in this way would be hard to discover,
+        maintain as DID docs evolved, and integrate into interoperable applications. DID methods that use
+        a public ledger or a similar source of truth are a better choice here, because they have authoritative
+        answers to the publication problem.
+    </p>
+</section>
+
+<section>
+    <h2><code>did:key</code> and <code>did:nacl</code></h2>
```

Comment:
Was @brentzundel or @ken-ebert able to bring you up to speed with the realization we've had wrt. did:key and did:peer? That they are really different because the protocols for resolving them are different, and because of that, it does make sense for them to be two different specs? If not, there it is... and this PR looks super aligned w/ that direction, which is great.

... now, all we have to do is align did:key and did:nacl *facepalm* -- this is why standards are hard... but really happy to see the direction w/ did:peer here. I may nitpick on some of the finer details, but super supportive of this direction in general.